src/app/containers/Include/amp/Idt2Amp.jsx

```diff
@@ -11,7 +11,7 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
 const Idt2Amp = ({ imageBlock }) =>
   imageBlock ? (
     <IncludeGrid>
-      <AmpImg fallback={false} {...imageBlock} />
+      <AmpImg {...imageBlock} />
```

Comment:
Does this remove the fallback image entirely?

Reply:
I guess my question is do we want a fallback image?

Reply:
The images themselves fallback images, I guess we'd have to have some generic fallback image when these aren't available? Thoughts @jimjohnsonrollings

Reply:
UX have designed a more generic fallback for when an include fails entirely on AMP... we should perhaps use that... but not today.  So I think it's fine today not to fall back?

Reply:
Sounds good @jimjohnsonrollings, is there an issue for this, I can create one otherwise?